src/XMLSerializer.php

```diff
@@ -5,12 +5,6 @@
 
 class XMLSerializer {
 
-    /** @var \XMLWriter */
-    private $writer;
-
-    /** @var Token */
-    private $previousToken;
```

Comment:
inlined this property, as we can use a local variable after we inlined the `addToken` method